l10n-pocket/en/pocket/about.ftl

```diff
@@ -0,0 +1,56 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev-pocket.allizom.org/about/
+
+
+pocket-about-us = About Us
+
+pocket-about-us-page-desc = { -brand-name-pocket } empowers people to keep their minds focused, stimulated, and nourished. We’re powered by a community of interesting people with curious minds.
+
+
+pocket-about-us-hi = Hi, we're { -brand-name-pocket }.
+pocket-about-powered-by-community-of-interesting-people = We’re powered by a community of interesting people with curious minds, and we empower people to keep their minds focused, stimulated, and nourished. Which makes us a refreshing change from other digital spaces.
+pocket-about-us-save-inspirations = Save your inspirations
+
+# Variables:
+#   $savetopocket (url) link to https://getpocket.com/save-to-pocket
+#   $addtopocket (url) link to https://getpocket.com/add
+pocket-about-if-something-catches-your-interest = If something catches your interest, there’s probably something to it. With our <a id="save-button-for-the-internet" href="{ $savetopocket }"> save button for the internet</a> and our award-winning <a id="top-pocket-app" href="{ $addtopocket }">{ -brand-name-pocket } app</a> you can save anything from across the web in a distraction free space, for you to focus and let the ideas flow.
```

Comment:
Suggestion: avoid including HTML attributes such as `id` in localized strings, as if there's a dependency for either functionality or styling things can break if a locale makes a mistake. Instead, attributes should be passed down as variables if they are required (in the same way we do for `href` values).

Reply:
Thanks - great catch. Fixing!

Reply:
Here's en axample where we pass down both an attribute and an href as a single variable. Maybe that might work here too https://github.com/mozilla/bedrock/blob/main/l10n/en/firefox/products.ftl#L58-L61

That said, I don't know why these links have attributes as such. Perhaps they are not needed and you can get rid of them? (I haven't checked yet but worth a look).